### ISSUE TITLE
Specify what happens with ibc_channel_close for randomness channels

### ISSUE THREAD

**Issue by webmaster128** [noislabs/nois-contracts#207]
Disallow closing? Doe something with remaining balance?

**Comment by kaisbaccour** [comment]
I think what we should have (maybe not a prio) is a way for the proxy (or maybe for gateway mnager) to transfer payment funds from a payment contract to another. this will be useful anytime a customer proxy wants to migrate the contract into a new address or change the chain_id. 
Another option is less automatic but good enough for now is that IBC_close will burn the payment funds with then sends  back the ash as proof of burn. with that the community will manually reimburse via governance the old proxy by refunding the new payment contract.

**Comment by webmaster128** [comment]
If you consider the dapp IBC counterparty on the other chain the ultimate customer and owner of the funds, we can just allow it to dens a TransferFunds IBC message on the same path it would usually use for RequestBeacon (see also #206). This should cover a lot of cases in which the channel and dapp contract is still healthy.

**Comment by kaisbaccour** [comment]
Yeah good question if you want to consider dapps owners of those NOIS or not. I think it should be the same way it works for KLM or Lufthansa miles you earn. They are yours, you can use them but you cannot sell them but If you change your account you should be able to call the support and transfer them to your new account.

**Comment by webmaster128** [comment]
Right, TransferFunds can be something very custom, not a generic send. It should not be attractive to hack a dapp to get the NOIS out. Worst case an attacker can use it for randomness.
